blog/2020-09-24-debezium-1-3-cr1-released.adoc

```diff
@@ -0,0 +1,69 @@
+
+As we approach the final stretch of Debezium 1.3 Final,
+we took this opportunity to add delegate converter support for the `ByteBufferConverter` and introduce a `debezium-scripting` module.
+In addition, there's also a range of bug fixes and quite a bit of documentation polish;
+overall, not less than https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.3.0.CR1%20ORDER%20BY%20issuetype%20DESC&startIndex=20[15 issues] have been resolved for this release.
+
+== ByteBufferConverter improvements
+
+In order to fix a recent report (https://issues.jboss.org/browse/DBZ-2396[DBZ-2396]),
+the `ByteBufferConverter` can now be configured to delegate event serialization to an additional converter.
+This delegation is necessary so that heartbeat, transaction metadata, and schema change events (if applicable) can be serialized.
+
+In order to use the Outbox event router SMT and the `ByteBufferConverter` with these event types,
+the connector configuration must be changed to reflect the delegate converter and its configurable options.
+
+As an example to use the Apache Kafka `JsonConverter` as a delegate with schemas disabled,
+the following configuration would need to be included in the connector:
+
+[source]
+----
+value.converter=io.debezium.converters.ByteBufferConverter
+value.converter.delegate.converter.type=org.apache.kafka.connect.json.JsonConverter
+value.converter.delegate.converter.type.schemas.enable=false
+----
+
+== Scripting module
+
+In this release, the SMTs for content-based routing and filtering that both use JSR 233 scripting engines have been moved out of `debezium-core` and into a separate artifact `debezium-scripting` (https://issues.jboss.org/browse/DBZ-2549[DBZ-2549]).
+Any connector that previous used these SMTs requires that the new artifact be added to the plug-in directories for those connector(s).
+
+When using the Debezium container image for Kafka Connect, set the environment variable `ENABLE_DEBEZIUM_SCRIPTING` to `true` to enable this feature.
+This change is done to allow the scripting functionality to be available only in environments with an apppropriately secured Kafka Connect configuration interface.
+
+== Misc. Features and Bug Fixes
+
+In addition, the community has completed the work on some other features and fixes, too:
+
+* Catch up streaming before snapshot may duplicate messages upon resuming streaming https://issues.jboss.org/browse/DBZ-2550[DBZ-2550]
+* Fix Quarkus datasource configuration for Quarkus 1.9 https://issues.jboss.org/browse/DBZ-2558[DBZ-2558]
+* Implement connection retry support for Oracle https://issues.jboss.org/browse/DBZ-2531[DBZ-2531]
+
+As always, please refer to the link:/releases/1.3/release-notes/#release-1.3.0-cr1[release notes] for the complete list of resolved issues as well as procedures for upgrading from earlier Debezium versions.
+
+Thank you so much to all the community members contributing to this release:
```

Comment:
Let's add Guillaume, too; he's a Hatter, but not Debezium, and makes it less awkward than having just one name there:

Thank you so much to Guillaume Smet and Grant Cooksey for their contributions to this release.